soroban-env-host/src/e2e_invoke.rs

```diff
@@ -529,15 +807,21 @@ impl<'a> SnapshotSource for StorageMapSnapshotSource<'a> {
         {
             Ok((Rc::clone(entry), *live_until_ledger))
         } else {
-            Err(Error::from_type_and_code(ScErrorType::Storage, ScErrorCode::InternalError).into())
+            Err(HostError::from((
```

Comment:
Why is this change being made? It shouldn't be observable but it does make me a little nervous.

Reply:
Just for the sake of consistency. This is a no-op and the end result is exactly the same, unless I'm missing something.

Reply:
Ok just wanted to make sure.